dll/win32/shell32/wine/shell32_main.c

```diff
@@ -130,16 +130,19 @@ LPWSTR* WINAPI CommandLineToArgvW(LPCWSTR lpCmdline, int* numargs)
         while (*s)
             if (*s++=='"')
                 break;
+        /* skip to the first argument, if any */
+        while (*s==' ' || *s=='\t')
+            s++;
     }
     else
     {
         /* The executable path ends at the next space, no matter what */
-        while (*s && *s!=' ' && *s!='\t')
+        while (*s && *s!=' ' && *s!='\t' && *s!='\r' && *s!='\n')
+            s++;
+        /* skip to the first argument, if any */
+        while (*s==' ' || *s=='\t' || *s=='\r' || *s=='\n')
```

Comment:
If this also accepts stuff like `'\v'` and `'\f'` you could probably use `isspace` here.

Reply:
OK.